src/Components/Patient/DailyRounds.tsx

```diff
@@ -195,36 +193,53 @@ export const DailyRounds = (props: any) => {
         setFacilityName("");
       }
       if (consultationId && !id) {
-        const res = await dispatchAction(
-          getDailyReport({ limit: 1, offset: 0 }, { consultationId })
+        const { data: dailyReportsData } = await request(
+          routes.getDailyReports,
+          {
+            pathParams: {
+              consultationId,
+            },
+            query: {
+              limit: 1,
+              offset: 0,
+            },
+          }
+        );
+        setHasPreviousLog(
+          (dailyReportsData && dailyReportsData?.count > 0) || false
         );
-        setHasPreviousLog(res.data.count > 0);
         formData = {
           ...formData,
           ...{
-            patient_category: res.data.patient_category
+            patient_category: dailyReportsData?.results[0].patient_category
               ? PATIENT_CATEGORIES.find(
-                  (i) => i.text === res.data.patient_category
+                  (i) =>
+                    i.text === dailyReportsData?.results[0].patient_category
                 )?.id ?? ""
               : "",
+
             rhythm:
-              (res.data.rhythm &&
-                RHYTHM_CHOICES.find((i) => i.text === res.data.rhythm)?.id) ||
+              (dailyReportsData?.results[0].rhythm &&
+                RHYTHM_CHOICES.find(
+                  (i) => i.text === dailyReportsData?.results[0].rhythm
+                )?.id) ||
               "0",
-            temperature: parseFloat(res.data.temperature),
+            temperature: dailyReportsData?.results[0].temperature
+              ? parseFloat(dailyReportsData?.results[0].temperature)
+              : null,
```

Comment:
We should not be copying these values unless copy from previous log checkbox is ticked right?